bip-0085.mediawiki

```diff
@@ -95,7 +95,36 @@ OUTPUT
 
 * DRNG(80 bytes)=b78b1ee6b345eae6836c2d53d33c64cdaf9a696487be81b03e822dc84b3f1cd883d7559e53d175f243e4c349e822a957bbff9224bc5dde9492ef54e8a439f6bc8c7355b87a925a37ee405a7502991111
 
-==Reference Implementation==
+==Changelog==
```

Comment:
Maybe ~follow the following order, per that of other BIPs:

- Abstract
- Copyright
- Definitions / Motivation

.../...

- References
- Reference Implementation
- Changelog
- Footnotes
- Acknowledgements

Reply:
@akarve what do you think?

Reply:
Sounds reasonable. Will put something together over the weekend.

Reply:
> @@ -95,7 +95,36 @@ OUTPUT * DRNG(80 bytes)=b78b1ee6b345eae6836c2d53d33c64cdaf9a696487be81b03e822dc84b3f1cd883d7559e53d175f243e4c349e822a957bbff9224bc5dde9492ef54e8a439f6bc8c7355b87a925a37ee405a7502991111 -==Reference Implementation== +==Changelog==